frontend/assets/stylesheets/components/_panels.scss

```diff
@@ -99,3 +99,20 @@
     padding: rem($gs-baseline * 2);
     margin-bottom: rem($gs-baseline * 2);
 }
+
+/* Challenge for member content */
+.challenge--panel {
```

Comment:
Normally `--` is used for a modifier class in BEM syntax. i.e. it would only be used in combination with a base class, e.g. `.challenge`. https://github.com/guardian/membership-frontend/blob/master/docs/client-side-principles.md#unintended-modifiers